### PR TITLE
feat: hide schemas in toc

### LINKED ISSUE #709: Show/hide "Schemas"
As an Elements user, I can pass a prop into the API component, so I can hide the "Schems" in the left navigation.

<p></p>

- Add a property `hideSchemas` to the API component

- When enabled, we should hide the "Schemas" part of the generated ToC

### PR DESCRIPTION
**Pull request by mpodlasin**
Closes #709

### CODE REVIEW

File: packages/elements/src/components/API/utils.ts [hunk 1]
```diff
@@ -51,7 +51,19 @@ export const computeTagGroups = (serviceNode: ServiceNode) => {
...
+
+const defaultComputerAPITreeConfig = {
+  hideSchemas: false,
+};
+
+export const computeAPITree = (serviceNode: ServiceNode, config: ComputeAPITreeConfig = {}) => {
+  const mergedConfig = {
+    ...defaultComputerAPITreeConfig,
+    ...config,
+  };
```
Comment (marcelltoth) [inline packages/elements/src/components/API/utils.ts:66]: This is a bit dangerous as a config like `{hideSchemas: undefined}` can override the default. Not a huge deal here as the default is `false`, but as a pattern in general, be aware of the problems. A safer tool is something like [_.defaults](https://lodash.com/docs/4.17.15#defaults) or similar property-by-property logic e.g. with a `??`.

File: packages/elements/src/components/API/__tests__/utils.test.ts [hunk 2]
```diff
@@ -326,3 +326,117 @@ describe('computeTagGroups', () => {
...
+        id: '/schemas/ImportantSchema',
+        slug: '/schemas/ImportantSchema',
+        title: 'ImportantSchema',
+        type: 'model',
+        meta: '',
+      },
+    ]);
+  });
+
+  it('hides schemas from ToC', () => {
```
Comment (marcelltoth) [inline packages/elements/src/components/API/__tests__/utils.test.ts:391]: *when instructed* or something. It doesn't hide them by default.

**Review by marcelltoth — APPROVED** [review]
Only that pattern below is concerning but also it happens to be correct in this case, so 👍.

File: packages/elements/src/components/API/utils.ts (hunk 1 above)
Comment (mpodlasin) [reply packages/elements/src/components/API/utils.ts]: I used defaults.

File: packages/elements/src/components/API/__tests__/utils.test.ts (hunk 2 above)
Comment (mpodlasin) [reply packages/elements/src/components/API/__tests__/utils.test.ts]: Yup, done 👍

**Pull request merged**